postgre-client/src/asset_filter_client.rs

```diff
@@ -262,14 +296,11 @@ fn add_filter_clause<'a>(
                 TokenType::All => {
                     query_builder.push(" AND (assets_v3.ast_owner = ");
                     query_builder.push_bind(owner_address);
-                    query_builder.push(" OR (fungible_tokens.fbt_owner = ");
-                    query_builder.push_bind(owner_address);
```

Comment:
looks like the below `if` should also be dropped. As well as the whole join on fungible_tokens. Also @n00m4d might be working on a concurrent task with balances. I believe with your change and dropped join Vadim task will also be solved.